test/unit/specs/components/datatable.spec.js

```diff
@@ -15,8 +15,8 @@ describe("DataTable.vue", () => {
         columns: [
           { name: "id", display_name: "Select Items", align: "center", checkbox: true },
           "name",
-          { name: "email", display_name: "Email Address", align: "center" },
-          { name: "age", datatype: "number", summary_value: "33" },
+          { name: "email", display_name: "Email Address", align: "center", sortable: true },
+          { name: "age", datatype: "number", summary_value: "33", sortable: true },
         ],
         jsonData: [
           { id: 1, name: "foo", email: "foo@xxx.xxx", age: 42 },
```

Comment:
Change this line to 4 and see where is sorts to validate my numeric sort question